docs/infra/api-markdown-documenter/render-api-documentation.mjs

```diff
@@ -141,13 +141,22 @@ export async function renderApiDocumentation(inputDir, outputDir, uriRootDir, ap
 			}
 			return alerts;
 		},
-		skipPackage: (apiPackage) => {
-			const packageName = apiPackage.displayName;
-			const packageScope = PackageName.getScope(packageName);
+		exclude: (apiItem) => {
+			// Exclude packages that aren't intended for public consumption.
+			if (apiItem.kind === ApiItemKind.Package) {
+				const packageName = apiItem.name;
```

Comment:
Asssuming `apiPackage.displayName -> apiItem.name` is correct?

Reply:
Ah, meant to leave a comment about this. Yes, `name` is a field package items specifically have. For package items, `displayName` yields the same thing, but _semantically_ name is more appropriate. (For other item kinds, `name` and `displayName` might actually yield different contents.